packages/csv-parser-discount-code/src/cli.js

```diff
@@ -6,6 +6,15 @@ import yargs from 'yargs'
 
 import CsvParserDiscountCode from './main'
 
+const doesFileExist = (filePath) => {
```

Comment:
Just a little observation, it will be nice if we can add a simple unit test to test this function.